assets/app/view/chat.rb

```diff
@@ -74,6 +74,7 @@ def render
     end
 
     def add_line(data)
+      data[:created_at] = Time.at(data[:created_at])
```

Comment:
why is this done here?

Reply:
Because this code was written in patches and it got confusing keeping the data structures consistent on the chat and on the actual log. But looking back it makes no sense to be here, because on sending a message it's converted to timestamp and back to object.

It's moved.